src/ls_pmmg.c

```diff
@@ -102,11 +104,26 @@ int PMMG_cuttet_ls(PMMG_pParMesh parmesh, MMG5_pMesh mesh, MMG5_pSol sol, MMG5_p
   int idx_face_ext,idx_face_int,val_face;
 
   if ( parmesh->info.imprim > PMMG_VERB_VERSION )
-    fprintf(stdout,"\n      ## PMMG_cuttet_ls: Multimaterial not supported yet.\n");
+    fprintf(stdout,"\n      ## PMMG_cuttet_ls: Multimaterial not fully supported yet.\n");
 
   /* Ensure only one group on each proc */
   assert(parmesh->ngrp == 1);
 
+  /* For now, does not support `nosplit` option in multimat  */
+  // To be removed when supported
+  for (i=0; i<mesh->info.nmat; i++) {
+    mat    = &mesh->info.mat[i];
+    ref    = mat->ref;
+    refint = mat->rin;
+    refext = mat->rex;
+    if ( (ref == refint) & (ref == refext) ) {
+      if ( parmesh->info.imprim > PMMG_VERB_VERSION ) {
+        fprintf(stdout,"\n      -- WARNING: The option `nosplit` in multimat is not supported yet.\n\n\n");
+        PMMG_RETURN_AND_FREE( parmesh, PMMG_LOWFAILURE );
+      }
```

Comment:
As the option is not supported, you always want to stop the process so I would always print an error message and exit, not only in case of `PMMG_VERB_VERSION` verbosity:
```
     fprintf(stderr,"\n      -- ERROR: The option `nosplit` in multimat is not supported yet.\n\n\n");
     PMMG_RETURN_AND_FREE( parmesh, PMMG_LOWFAILURE );
```

Reply:
Done + modify return to be `0` instead of `PMMG_RETURN_AND_FREE` (here and elsewhere in the file too).